site_scons/components/__init__.py

```diff
@@ -172,7 +172,19 @@ def define_mercury(reqs):
         MERCURY_DEBUG = '-DMERCURY_ENABLE_DEBUG=OFF '
     retriever = \
         GitRepoRetriever('https://github.com/mercury-hpc/mercury.git',
-                         True)
+	True)
+
+    reqs.define("UCX", libs=['ucp'])
```

Comment:
Every other component uses lower case.  My personal preference would be to be consistent and just call it ucx but this is a minor point.